### PR TITLE
Logging weights, hyperparameters and metrics

### PR DESCRIPTION
**Pull request by Raalsky**

### CODE REVIEW

File: examples/tabular_model.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
+from neptune import new as neptune
+from neptune_fastai.impl import NeptuneCallback
+
+from fastai.basics import *
```
Comment (shnela) [inline examples/tabular_model.py:4]: I'm not familiar with data scientist practices and libraries, but I'm quite aware that it's a bad idea.

We're using `import *` in one place of our code (importing of our `neptune-<integration>` libraries), but we consciously define `__all__` in our integrations.

File: neptune_fastai/_version.py [hunk 2]
```diff
@@ -1,3 +1,4 @@
+# pylint: skip-file
```
Comment (shnela) [inline neptune_fastai/_version.py:1]: What's wrong with this file?

Comment (shnela) [reply neptune_fastai/_version.py:1]: Maybe we should disable something in `pylintrc`?

**Review by shnela — CHANGES_REQUESTED** [review]

File: neptune_fastai/_version.py (hunk 2 above)
Comment (Raalsky) [reply neptune_fastai/_version.py:1]: Currently I don't have an output of pylint but it also disabled in other integrations for ex.: https://github.com/neptune-ai/neptune-tensorflow-keras/blob/master/neptune_tensorflow_keras/_version.py

File: examples/tabular_model.py (hunk 1 above)
Comment (Raalsky) [reply examples/tabular_model.py:4]: I just copy-paste official examples from fastai :wink:. I 100% agree that it might be confusing.

**Pull request merged**